src/main/java/org/jenkinsci/plugins/stashNotifier/StashNotifier.java

```diff
@@ -449,7 +449,7 @@ public DescriptorImpl getDescriptor() {
     }
 
     @Extension
-	public static final class DescriptorImpl
+	public static class DescriptorImpl
```

Comment:
I'm not sure we should unseal this class. I've not noticed any extensions with your patch